src/main/java/org/minions/devfund/marcos/uml/Door.java

```diff
@@ -0,0 +1,79 @@
+package org.minions.devfund.marcos.uml;
+
+/**
+ * It is in charged to represent a Door object.
+ */
+public class Door extends MapSite {
+
+    private boolean isOpen;
+    private Room room1;
+    private Room room2;
+
+    /**
+     * Initializes an instance of {@link Door}.
+     */
+    public Door() {
+        // Default constructor.
+    }
+
+    /**
+     * Enters to room.
+     */
+    public void enter() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1161)